django/applications/catmaid/templates/catmaid/index.html

```diff
@@ -51,7 +51,9 @@
     {% endfor %}
 
     {% for f in NON_COMPRESSED_FILES %}
-      <script type="application/javascript" src="{{ STATIC_URL }}{{ f }}"></script>
+      {% if 'neuroglancer' not in f %}
```

Comment:
I think it would be better if we don't have tool-specific exceptions here. It is probably worthwhile generalizing this into a `MANUALLY_LOADED_FILED` or something similar. Files listed through this variable, won't be included in the front-end template.